Changes

```diff
@@ -1,5 +1,11 @@
 {{$NEXT}}
 
+- Added support for 'average_income' and 'estimated_population' in the
+  Insights location record
```

Comment:
We should do a minor version bump as opposed to the default patch bump as we are ostensibly following Semantic Versioning for this dist.